### ISSUE TITLE
[Feature Request] Ability to store an incoming-webhook url.

### ISSUE THREAD

**Issue by avelis** [izdi/django-slack-oauth#4]
Slack supports the ability to request an access scope of `incoming-webhook` when authenticating for an access token.

This would in turn provide a response payload as follows:

```
{
    "access_token": "xoxp-XXXXXXXX-XXXXXXXX-XXXXX",
    "scope": "incoming-webhook"
    "team_name": "Team Installing Your Hook",
    "incoming_webhook": {
        "url": "https://hooks.slack.com/TXXXXX/BXXXXX/XXXXXXXXXX",
        "channel": "#channel-it-will-post-to",
        "configuration_url": "https://teamname.slack.com/services/BXXXXX"
    }
}
```

https://api.slack.com/docs/slack-button (See Step 4)

While the access token is already stored, the web-hook url is not.

Understanding the current implementation state of this library, this will likely require a migration and modifications to:

SlackAuthView: https://github.com/izdi/django-slack-oauth/blob/master/django_slack_oauth/views.py#L35

Thoughts?

**Comment by izdi** [comment]
I'd like to add `extras` field `JSONField()` which will contain json body for all requests

**Comment by avelis** [comment]
@izdi Do want an `extras` field along side a `webhook_url` field? Or would the `extras` field replace it?

**Comment by izdi** [comment]
I provide solution today

**Comment by izdi** [comment]
Fixed in a0c879d27d888bdbb151b450e5c6c5a711fc7084
